suricata/update/main.py

```diff
@@ -509,6 +509,13 @@ def write_to_directory(directory, files, rulemap, dep_files):
     modified = []
 
     oldset = {}
+
+    cookie_path = os.path.join(directory, ".suricata-update")
+    if os.path.exists(directory):
+        if not os.path.exists(cookie_path) and os.listdir(directory):
+            logger.error("{} -- doesn't look like a suricata-update rule directory".format(directory))
+            sys.exit(1)
```

Comment:
@jasonish do you think this is as per your expectations?
Questions:
1. How do we ensure that this does not break the current setups? Because we don't have a cookie already in the rule dir.
2. Is checking for dir good enough for determining  "first run"?
3. Should the cookie be in say `<prefix>/suricata` or `<prefix>/suricata/rules`